src/utils.ts

```diff
@@ -24,6 +31,26 @@ export function exit (message: string): never {
   Deno.exit(1)
 }
 
+export async function getBackend (src: string): Promise<Backend> {
```

Comment:
This function seems to be only referenced from one place at the moment. Does it make sense to call it from more places to simplify code?

Reply:
Yeah likely, but it's a bit complicated because this function only looks at the `src` string to find a backend whereas the command line for e.g. `chel migrate` specifies both the backend type e.g. `sqlite` and the source.

Reply:
I've opened up an issue related to this: https://github.com/okTurtles/chel/issues/20